Mobile/ShopLens/MigrationsConsole/README.md

```diff
@@ -0,0 +1,20 @@
+## Working with DB
+
+The *ShopLensWeb* project contains all the models and db contexts that you need.
+
+The *MigrationsConsole* project has links to those models. This way
+migrations can be produced from a simple console application. From this 
+project you can also write queries, fill the DB with data, etc.
+
+### Using the local DB
+
+Set the "UseLocalDb" flag in the App.config of *MigrationsConsole* to "FALSE".
+Pass *true* to the ShopLensContext's *useLocalDb* parameter.
```

Comment:
This is not right at all. You just set the flag to "TRUE" and use the local database. You don't need to pass true explicitly. I don't understand why is this README written so.

Reply:
You just need to pass the value you got from the app.config.

Reply:
Yeah, I messed up, it should be the other way around. 

Good catch :ok_hand: 

Fixed it